wandb/data_types.py

```diff
@@ -0,0 +1,31 @@
+class Histogram(object):
+    MAX_LENGTH = 512
+
+    def __init__(self, list_or_tuple, num_bins=64):
+        """Accepts a tuple of (values, bins_edges) as np.histogram returns i.e.
+
+        wandb.log({"histogram": wandb.Histogram(np.histogram(data))})
+
+        Or a list of values in which case they will be automatically binned into num_bins.
+        The maximum number of bins currently supported is 512
+        """
+        if isinstance(list_or_tuple, tuple) and len(list_or_tuple) == 2:
```

Comment:
I think use this to check if it's a sequence: isinstance([], collections.Sequence)

And then here check if it's a sequence of length 2, with each inner element being a sequence.